CMakeLists.txt

```diff
@@ -443,6 +453,9 @@ endif()
 include(CMakePackageConfigHelpers)
 configure_package_config_file(sentry-config.cmake.in sentry-config.cmake
 	INSTALL_DESTINATION "${CMAKE_INSTALL_CMAKEDIR}")
+write_basic_package_version_file(sentry-config-version.cmake 
+	VERSION ${SENTRY_VERSION}
+	COMPATIBILITY AnyNewerVersion)
```

Comment:
Haven’t looked at the docs for this yet, but is **any** semver compatible?

Reply:
for example, `find_package(sentry 1.2.3)`

* `AnyNewerVersion` --> any version >= 1.2.3
* `SameMajorVersion` --> 1.x.x
* `SameMinorVersion` --> 1.2.x
* `ExactVersion` --> 1.2.3

Reply:
I think in that case we should opt for `SameMinorVersion` as long as we are `< 1.0.0`

Reply:
changed, with a reminder for >=1.0.0